marketplace_confg/schema.yaml

```diff
@@ -27,129 +27,162 @@ properties:
+      (Optional - Redactor) Tags to remove during C-STORE upload, comma 
+      separated. Only one of 'redact' flags may be present.
   import.redact_keep_list:
     type: string
-    title: "importer - redact keep list (Optional)"
+    title: "importer - Redact keep list (Optional)"
     description:  >-
-      "(Optional - Redactor) Tags to keep during C-STORE upload, comma separated.
-      Only one of 'redact' flags may be present."
+      (Optional - Redactor) Tags to keep during C-STORE upload, comma separated.
+      Only one of 'redact' flags may be present.
   import.dimse_cmove_aet:
     type: string
     title: "importer - C-MOVE aet (Optional)"
     description:  >-
-      "(Optional - C-MOVE) Separate AET used for C-STORE calls within context of C-MOVE."
+      (Optional - C-MOVE) Separate AET used for C-STORE calls within context of C-MOVE.
+      For details, go to:
+      https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter#import-adapter
   import.aet_dictionary_inline:
     type: string
     title: "importer - Inline aet dictionary (Optional)"
     description:  >-
-      "(Optional - C-MOVE) Json array containing aet definitions (name/host/port per element)"
+      (Optional - C-MOVE) Json array containing aet definitions (name/host/port per element).
+      e.g. [{"name":"DEVICE_A","host":"localhost","port":11113},
+      {"name":"DEVICE_B","host":"192.168.0.1","port": 11114},...]
+      For details, go to:
+      https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter#import-adapter
   import.destination_config_inline:
     type: string
-    title: "importer - destination config inline (Optional)"
+    title: "importer - Destination config inline (Optional)"
     description:  >-
-      "(Optional) Json array containing destination definitions 
-      (filter/dicomweb_destination per element)."
+      (Optional) Json array containing destination definitions 
+      (filter/dicomweb_destination per element).
   import.store_compress_to_transfer_syntax:
     type: string
-    title: "importer - store compress to transfer syntax (Optional)"
+    title: "importer - Store compress to transfer syntax (Optional)"
     description:  >-
-      "(Optional) Transfer Syntax to convert instances to during C-STORE 
-      upload. See Readme for list of supported syntaxes."
+      (Optional) Transfer Syntax to convert instances to during C-STORE 
+      upload. See Readme for list of supported syntaxes.
+  import.monitoring_project_id:
+      type: string
+      title: "importer - Monitoring project id (Optional)"
+      description: >-
+        (Optional) Stackdriver monitoring project id, must be the same as the 
+        project id in which the adapter is running.
+  import.custom_flags:
+      type: string
+      title: "Extra import adapter flag (Optional)"
+      description: >-
+        (Optional) Addtional flag for import adapter.
+        e.g. "--flag_a=valueA"
```

Comment:
can we add an example for multiple flags, to clarify if they have to be comma or space separated?

Reply:
Unfortunately the marketplace does not support list as input. I tried to input a list in the testing but it failed because it was recognize as a list rather than a string (even the list was in a string format)
In addition, Helm chart does not support string split or read string line by line. If the adapter accepts json string as argument I can add some examples here.

Reply:
Got it, ok then it's fine as-is.